sleap/skeleton.py

```diff
@@ -56,8 +56,8 @@ class Node:
         weight: Weight of the node (not currently used).
     """
 
-    name: str
-    weight: float = 1.0
+    name: str = attr.ib(validator=attr.validators.instance_of(str))
+    weight: float = attr.ib(default=1.0)
```

Comment:
Not necessary since `auto_attribs=True` in the decorator should take care of defining the `attr.ib`s?

Reply:
you are right, reverted back to original ...